assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SuccessBanner.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * SuccessBanner component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import BannerNotification from '../../../../../components/notifications/BannerNotification';
+
+export default function SuccessBanner() {
+	return (
+		<BannerNotification
+			id="ga4-activation-banner"
+			title={ __(
+				'You successfully set up your Google Analytics 4 property',
+				'google-site-kit'
+			) }
+			dismiss={ __( 'Ok, got it', 'google-site-kit' ) }
```

Comment:
Elsewhere in the plugin we seem to use a variation on this string:

<img width="265" alt="CleanShot 2022-08-04 at 16 29 28" src="https://user-images.githubusercontent.com/90871/182886669-efd44c88-3888-4afa-a7db-1a3e39804cb9.png">

This one isn't used anywhere, so let's not add another very similar string—it will require another translation because of the different casing/punctuation.

```suggestion
			dismiss={ __( 'OK, got it!', 'google-site-kit' ) }
```

Honestly it's probably best to replace all of these strings as well. I'll file an issue for it, but for now let's just use one of the existing ones.

Reply:
Saw the new issue you filed! Thanks!